google-cloud-firestore/test/google/cloud/firestore/client/transaction_test.rb

```diff
@@ -117,6 +119,21 @@
     end
   end
 
+  it "runs a simple query with read only transaction" do
+    expected_query = Google::Cloud::Firestore::V1::StructuredQuery.new(
+      select: Google::Cloud::Firestore::V1::StructuredQuery::Projection.new(
+        fields: [Google::Cloud::Firestore::V1::StructuredQuery::FieldReference.new(field_path: "name")]),
+      from: [Google::Cloud::Firestore::V1::StructuredQuery::CollectionSelector.new(collection_id: "users", all_descendants: false)]
+    )
+    firestore_mock.expect :run_query, query_results_enum, run_query_args(expected_query, new_transaction: read_only_transaction_opt)
+    # firestore_mock.expect :commit, empty_commit_resp, commit_args(transaction: transaction_id)
+
+    firestore.read_only_transaction read_time: read_time do |tx|
```

Comment:
Is `read_time` ignored in this case? Why isn't it in the mock expectation?

Reply:
No the `read_time` is not ignored, `read_only_transaction_opt` contains the `read_time` which is in the mock expectation